README.md

```diff
@@ -42,17 +42,6 @@ ARGUMENTS
 ---------
 <!-- TODO replace this with output from command line --help -->
 
---deposits-dir | -d (optional)
```

Comment:
Fine, should we have a default option of the `deposits-dir`, like we have in the `springfield-inbox`? Not sure about that. Currently this `deposit-dir` is optional, so either we should have a fallback or we need to change this into a required field.